src/types/nitro.ts

```diff
@@ -139,6 +140,7 @@ export interface NitroOptions {
   externals: NodeExternalsOptions
   analyze: false | PluginVisualizerOptions
   replace: Record<string, string | ((id: string) => string)>
+  commonJs?: RollupCommonJSOptions
```

Comment:
Can you please use `commonJS` for naming?

Reply:
done \:+1: